bin/update-pdfjs

```diff
@@ -9,9 +9,10 @@ set -eu
 #
 # To update PDF.js to the latest version:
 #
-# 1. Create a new branch and run this script.
-# 2. Verify that PDFs work as expected in py-server.
-# 3. Commit the changes to the `vendor/` and `templates/` directories.
+# 1. Create a new branch and run `make update-pdfjs`. This will download the
+#    newest release of PDF.js, update the HTML template for the viewer and
+#    commit the changes.
```

Comment:
I'm a little concerned that this script looks so similar to that in other projects that devs might "read past" this different instruction. What "happens" if this script is run directly instead of with `make update-pdfjs`. Is it possible to build in a safety check?

Reply:
Yes. I have pushed a commit that adds a check, using the presence of the `TOX_ENV_NAME` var to test if the script is being run directly or via tox.